components/layout/header/source-links.tsx

```diff
@@ -12,26 +12,33 @@ const SourceLinks = (props) => (
     icon={<AiOutlineGithub size={24} style={{ marginBottom: -5 }} />}
     title="Source code"
   >
-    <ItemGroup title="Project">
+    <ItemGroup title="Projects">
       <Item icon={<AiOutlineGithub />} key="source:documentation">
         <a target="_blank" rel="noreferrer" href="https://github.com/game-ci/documentation">
           This Website
         </a>
       </Item>
+      <Item icon={<AiOutlineGithub />} key="source:game-ci">
+        <a target="_blank" rel="noreferrer" href="https://github.com/game-ci/versioning-backend">
+          Versioning Backend
+        </a>
+      </Item>
+    </ItemGroup>
+    <ItemGroup title="Docker">
+      <Item icon={<AiOutlineGithub />} key="external:unityci:editor">
+        <a target="_blank" rel="noreferrer" href="https://github.com/game-ci/docker">
+          Docker images for Unity
+        </a>
+      </Item>
+    </ItemGroup>
+    <ItemGroup title="Gitlab">
       <Item icon={<AiOutlineGitlab />} key="external:example:gitlab">
         <a
           target="_blank"
           rel="noreferrer"
           href="https://gitlab.com/gableroux/unity3d-gitlab-ci-example"
```

Comment:
This url is already fixed here: https://github.com/game-ci/documentation/pull/246 so I did not change it a second time in this pr.